easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.3.0-2.28-LAPACK-3.7.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
```

Comment:
This is the most recent, but relatively old, version (released Sept'16).

I reached out to the OpenBLAS maintainer (@xianyi), this was his reply: `We will release OpenBLAS in one or two weeks.`

So, probably worth waiting for...